scipy/signal/_spectral_py.py

```diff
@@ -1489,7 +1489,7 @@ def istft(Zxx, fs=1.0, window='hann', nperseg=None, noverlap=None, nfft=None,
 
     # Divide out normalization where non-tiny
     if np.sum(norm > 1e-10) != len(norm):
-        warnings.warn("NOLA condition failed, STFT may not be invertible")
+        warnings.warn("NOLA condition failed, STFT may not be invertible. " + ("Possibly due to missing boundary" if boundary else ""))
```

Comment:
This line is too long.
```suggestion
        warnings.warn(
            "NOLA condition failed, STFT may not be invertible."
            + ("Possibly due to missing boundary" if boundary else "")
        )
```